plugins/base/src/main/kotlin/resolvers/local/DokkaLocationProvider.kt

```diff
@@ -130,7 +139,7 @@ open class DokkaLocationProvider(
     private val PageNode.pathName: String
         get() = if (this is PackagePageNode) name else identifierToFilename(name)
 
-    protected data class DRIWithSourceSet(val dri: DRI, val sourceSet: DisplaySourceSet?)
+    protected data class DRIWithSourceSet(val dri: DRI, val sourceSet: Set<DisplaySourceSet>)
```

Comment:
Do `DRIWithSourceSet` rename to `DRIWithSourceSet`**s** similarly `withSourceSets` function?